### DISCUSSION TITLE
Lesson 7: Purpose of extending StdCheats in the tests in the repo

### DISCUSSION THREAD

**Discussion by ShayanShamsi** [Cyfrin/foundry-full-course-cu#117] [Q&A] [answered]
Hello! 
In [the repo](https://github.com/Cyfrin/foundry-fund-me-f23/blob/main/test), `FundMeTest.t.sol` and `InteractionsTest.t.sol` are extending `StdCheats` which was not done in the YouTube video. Can someone please explain what is the purpose of that? I do not see any significant differences compared to the code from the YouTube video in the rest of the code so I am unable to figure out the purpose of extending `StdCheats`.

**Comment by PatrickAlphaC** [comment]
Try it with and without and see if it makes a difference ;)

**Comment by alymurtazamemon** [comment, marked as answer]
@ShayanShamsi It is Wrappers around Forge cheat codes for improved safety and DX.

For more information [Forge Std's Test](https://book.getfoundry.sh/reference/forge-std/#forge-stds-test)
